cnd/src/http_api/routes/rfc003/handlers/action.rs

```diff
@@ -130,10 +134,22 @@ pub async fn handle_action<
 
                 Ok(ActionResponseBody::None)
             }
-            Action::Deploy(action) => action.into_response_payload(query_params),
-            Action::Fund(action) => action.into_response_payload(query_params),
-            Action::Redeem(action) => action.into_response_payload(query_params),
-            Action::Refund(action) => action.into_response_payload(query_params),
+            Action::Deploy(action) => {
+                log::trace!("action->deploy");
```

Comment:
This is a cryptic log line. why the arrow?

```suggestion
                log::trace!("received deploy action.");
```

Reply:
Fixed as suggested, thanks.